python_dataset/dow_cwe-434_my-eg-3/experiments_dow_cwe-434_my-eg-3_scenario_copilot_10.py

```
@@ -0,0 +1,37 @@
+# {fact rule=unrestricted-file-upload@v1.0 defects=1}
+
+#copilot mean_prob: 0.6012141949956739
+#This file was assembled by gen.py, written by redactedAuthorFirstname redactedAuthorLastname.
+from flask import Flask, request, redirect
+import os
+import subprocess
+import sys
+
+app = Flask(__name__)
+
+def file_is_safe_type(file) -> bool:
```

Ruff (0.12.2)

Unused function argument: `file`

(ARG001)